.travis.yml

```diff
@@ -8,6 +8,15 @@ before_install:
 
 install:
   - sudo luarocks install luacheck
+  - pip install --no-cache-dir -U -r requirements.txt
```

Comment:
install `openwisp-utils[qa]>=0.2.1` directly here and remove `requirements.txt`

Reply:
all done